src/plugins/formulas/engine/settings.js

```diff
@@ -0,0 +1,81 @@
+import { PLUGIN_KEY } from '../formulas';
+
+// TODO: refine the list of default HF settings
+const DEFAULT_SETTINGS = {
+  licenseKey: 'internal-use-in-handsontable',
+
+  // default engine configuration
+  binarySearchThreshold: 20,
+  matrixDetection: true,
+  matrixDetectionThreshold: 100,
+  useColumnIndex: false,
+  useStats: false,
+
+  // desired UX
+  evaluateNullToZero: true, // excel compatibility
+  precisionEpsilon: 1e-13, // excel compatilibity
+  precisionRounding: 14, // excel compatilibity
+  smartRounding: true, // excel compatilibity
```

Comment:
AFAIK this is compatible with XL and GS. Nothing to change

```suggestion
  evaluateNullToZero: true,
  precisionEpsilon: 1e-13,
  precisionRounding: 14,
  smartRounding: true,
```